openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/DescriptionSuggestion.spec.ts

```diff
@@ -25,7 +25,7 @@ const user2 = new UserClass();
 const user3 = new UserClass();
 let entityLinkList: string[];
 
-test.describe('Description Suggestions Table Entity', () => {
+test.describe.serial('Description Suggestions Table Entity', () => {
```

Comment:
why we are adding serial here?

Reply:
The way tests are written, serial execution is necessary to avoid the varying of expected suggestion counts based on which test is executed first.